Tryouts/Messaging-JS/.vscode/templates/js.lict

```diff
@@ -0,0 +1,9 @@
+Morgan Stanley makes this available to you under the Apache License,
```

Comment:
Should we add lict files for the .NET part too?

Reply:
We can, but I've been adding the license headers with ReSharper, and they are not compatible. I'm not sure if the vscode extension I used here is something widely used or just a random first hit.